frontend/Controls/UnfinishedModuleMenuButton.qml

```diff
@@ -0,0 +1,24 @@
+import QtQuick 2.7
+import "../Theme" 1.0
+
+ModuleMenuButton {
+    MouseArea {
+        id: mouseArea
+        anchors.fill: parent
+        cursorShape: Qt.PointingHandCursor
+
+        onClicked: {
+            modalAlert({
+                           bodyText: "This portion of XCite is not yet functioning, expect it soon",
+                           title: qsTr("Module Alert"),
+                           buttonText: qsTr("OK")
+                       })
+            selectView(target)
+        }
+
+        hoverEnabled: true
+        onHoveredChanged: {
+            button.state = containsMouse ? "hover" : ""
```

Comment:
This causes errors for all uses: qrc:/Controls/UnfinishedModuleMenuButton.qml:21: ReferenceError: button is not defined - could you fix that up please?